src/components/Header/containers/HeaderContainer.js

```diff
@@ -1,22 +1,11 @@
 import { connect } from 'react-redux';
 
 import Header from '../Header';
-import * as selectors from '../../../selectors';
-import * as appActions from '../../../actions';
 
-function mapStateToProps(state, ownProps) {
+function mapStateToProps() {

```

Comment:
For this case, you don't need to have an container. You only have one property and it's already hard coded set.

Reply:
Okay, this fix will be available along with the next task's PR.